.github/workflows/run_tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       DJANGO_ENV: "TESTING"
     strategy:
       matrix:
-        python: ["3.8"]
+        python: ["3.10"]
```

Comment:
I think we're still running Python 3.8 in production, so please keep the two identical. Most type hints are already available in 3.8. E.g. `List[str]` (with an import for `List` from Python's `typing` module) opposed to `list[str]`

Reply:
Yeah I already reverted this before you commented, but thanks for the feedback!